src/test/java/com/github/tomakehurst/wiremock/jetty11/JettyHttpServerTest.java

```diff
@@ -119,4 +124,17 @@ public void testHttpConnectorIsNullWhenHttpDisabled()
 
     assertNull(httpConnector);
   }
+
+  @Test
+  public void testStartWithBindException() {
```

Comment:
I don't think this test case really proves anything.

An alternative would be to:
1. Add an info log statement indicating that a retry is happening and the count value.
2. Create a server socket on port 0
3. Try to start WireMock with this socket's port number and with verbose logging enabled
4. Assert that 3 retry log messages were written
5. Assert that the bind exception is thrown